### PR TITLE
feat: the kanban view supports full-field stacking

### PR DESCRIPTION
**Pull request by Sky-FE**

### CODE REVIEW

File: apps/nextjs-app/src/features/app/blocks/view/kanban/components/KanbanStack.tsx [hunk 1]
```diff
@@ -152,7 +147,7 @@ export const KanbanStack = forwardRef<VirtuosoHandle, IKanbanStackProps>((props,
                   draggableId={card.id}
                   index={realIndex}
                   key={card.id}
-                  isDragDisabled={!cardDraggable}
+                  isDragDisabled={!cardDraggable || isComputed}
```
Comment (tea-artist) [inline apps/nextjs-app/src/features/app/blocks/view/kanban/components/KanbanStack.tsx:150]: Is the field read-only permission not considered here?

File: apps/nextjs-app/src/features/app/blocks/view/kanban/components/KanbanStackHeader.tsx [hunk 2]
```diff
@@ -38,10 +38,10 @@ export const KanbanStackHeader = (props: IKanbanStackHeaderProps) => {
   const { permission, stackField } = useKanban() as Required<IKanbanContext>;
   const { t } = useTranslation(tableConfig.i18nNamespaces);
 
-  const { type, options } = stackField;
+  const { type, options, isLookup } = stackField;
   const { id: stackId, data: stackData } = stack;
   const { stackEditable, stackDeletable } = permission;
-  const isSingleSelectField = type === FieldType.SingleSelect;
+  const isSingleSelectField = type === FieldType.SingleSelect && !isLookup;
```
Comment (tea-artist) [inline apps/nextjs-app/src/features/app/blocks/view/kanban/components/KanbanStackHeader.tsx:44]: isLookup may also be single, isMultipleCellValue should be used

File: apps/nextjs-app/src/features/app/blocks/view/kanban/context/KanbanProvider.tsx [hunk 3]
```diff
@@ -80,7 +72,9 @@ export const KanbanProvider = ({ children }: { children: ReactNode }) => {
         : getBaseCollaboratorList(queryKey[1], queryKey[2] as ListBaseCollaboratorRo).then(
             (data) => data.data
           ),
-    enabled: Boolean((shareId || baseId) && stackFieldType === FieldType.User),
+    enabled: Boolean(
+      (shareId || baseId) && type === FieldType.User && !isMultipleCellValue && !isLookup
```
Comment (tea-artist) [inline apps/nextjs-app/src/features/app/blocks/view/kanban/context/KanbanProvider.tsx]: ditto

File: apps/nextjs-app/src/features/app/blocks/view/kanban/components/KanbanStack.tsx (hunk 1 above)
Comment (Sky-FE) [reply apps/nextjs-app/src/features/app/blocks/view/kanban/components/KanbanStack.tsx:150]: `cardDraggable` indicates whether it has read-only permissions.

File: apps/nextjs-app/src/features/app/blocks/view/kanban/components/KanbanStackHeader.tsx (hunk 2 above)
Comment (Sky-FE) [reply apps/nextjs-app/src/features/app/blocks/view/kanban/components/KanbanStackHeader.tsx:44]: Currently, filtering does not support distinguishing between single and multiple values for lookup fields.

**Review by tea-artist — APPROVED** [review]

**Pull request merged**